browser/search.mjs

```diff
@@ -28,7 +28,17 @@ const inputRef = createRef()
 const root = document.getElementById('issue-search')
 root.innerHTML = ''
 
-const search = (state, push) => {
+const tokenIcon = byKey(
+  {
+    '#': '🏷️',
+    '@': '🧑',
+    '"': '🔤',
```

Comment:
what's the quote for?

Reply:
The query language supports `"` for tokens with spaces. This allows configuring an icon for the token button. 

![2023-12-19-191411_424x203_scrot](https://github.com/bas080/issue/assets/1840380/a7ffe5ff-f239-4a34-9ccb-a66372c91dc6)

Reply:
its a bit confusing to have an opening but no closing bracket